src/main/java/uk/nhs/hee/tis/trainee/forms/api/FeatureFlagResource.java

```diff
@@ -49,6 +49,15 @@ public class FeatureFlagResource {
   @GetMapping
   public ResponseEntity<FeatureConfigurationProperties> getFeatureFlags() {
     log.debug("Get all the feature flags for forms");
+
+    // Create a new instance of FormRPartB and set covidDeclaration to false
+    FeatureConfigurationProperties.FormRPartB formRPartB =
+        new FeatureConfigurationProperties.FormRPartB();
+    formRPartB.setCovidDeclaration(false);
+
+    // Update the featureConfig object with the new FormRPartB instance
+    featureConfig.setFormRPartB(formRPartB);
+
```

Comment:
You shouldn't be modifying the config like this. The value is set in the application properties https://github.com/Health-Education-England/tis-trainee-forms/blob/main/src/main/resources/application.yml#L12

We could just set the env var to override it, but the default should probably be changed to false instead in this case.